src/components/Modal/BaseModal.tsx

```diff
@@ -1,5 +1,7 @@
 import React, {forwardRef, useCallback, useContext, useEffect, useMemo, useRef} from 'react';
-import {View} from 'react-native';
+// Animated required for side panel navigation
```

Comment:
Why do we need this? Is there any plan to use something that is not restricted?

Reply:
we use an animated value obtained from navigation to fix the modals being covered by the help modal. unfortunately, react-navigation uses Animated, instead of Reanimated, under the hood, so we cannot escape it here :/